src/main/java/no/nav/veilarboppfolging/service/KafkaConsumerService.java

```diff
@@ -81,9 +81,6 @@ public void consumeEndringPaOppfolgingBruker(ConsumerRecord<String, EndringPaaOp
             log.info("Endring på oppfølgingsbruker fra Arena er eldre enn sist lagret endring. " +
                     "Dersom vi ikke utførte en rewind på topicen betyr dette at Arena har en uventet oppførsel. " +
                     "Denne loggmeldingen er kun til informasjon slik at vi eventuelt kan fange opp dette scenariet til ettertid.");
-            if (unleashService.skalIgnorereGamleEndringerFraVeilarbarena()) {
```

Comment:
are we still missing return here?

Reply:
The `return`-statement on line #85 only makes sense given that the feature-toggle still exists.